src/services/car/__tests__/inventory.test.ts

```diff
@@ -329,6 +330,44 @@ describe("CAR service", () => {
     })
   })
 
+  describe("#bulkArchiveDealerListing", () => {
+    it("archives listings", async () => {
+      fetchMock.mockResponse(JSON.stringify({ ok: true }))
+
+      const response = await bulkArchiveDealerListings({
+        dealerId: 6,
+        listingIds: [123, 124],
+        options: requestOptionsMock,
+      })
+      expect(response).toEqual({ tag: "success", result: {} })
+      expect(fetch).toHaveBeenCalledWith(
+        expect.stringContaining("/dealers/6/listings/archive"),
+        expect.objectContaining({ method: "POST" })
```

Comment:
would be nice to also expect the actual body that the backend expects in order to have that also under test :)